packages/hash/frontend/src/components/hooks/blockProtocolFunctions/ontology/useBlockProtocolAggregateEntityTypes.ts

```diff
@@ -59,7 +60,11 @@ export const useBlockProtocolAggregateEntityTypes = (): {
       }
 
       return {
-        data: response.data.getAllLatestEntityTypes,
+        /**
+         * @todo: remove this when we start returning links in the subgraph
```

Comment:
Because the frontend code is all using the same `Subgraph` object, which is the patched one (so we don't have mixed ones with confusing imports and hard to find errors).

Even though this doesn't contain any `EntityVertex`, the object allows for them so that's why the cast